src/pytest_html/scripts/dom.js

```diff
@@ -57,10 +57,10 @@ const dom = {
         }
 
         if (collapsed) {
-            resultBody.querySelector('.collapsible > td')?.classList.add('collapsed')
+            resultBody.querySelector('.collapsible > col-result')?.classList.add('collapsed')
```

Comment:
I don't have the code infront of my right now as I'm on vacation, but are we sure this will uniquely identify the column?

I ask because IIRC other things can have at least the ".collapsible" class.

Reply:
I believe it is but better if you can double check when you return from vacation.
Thanks.